third-party/chpl-venv/Makefile.include

```diff
@@ -35,3 +37,6 @@ CHPL_VENV_VIRTUALENV_BIN=$(CHPL_VENV_VIRTUALENV_DIR)/bin
 CHPL_VENV_SPHINX_BUILD=$(CHPL_VENV_VIRTUALENV_BIN)/sphinx-build
 CHPL_VENV_TEST_REQS=$(CHPL_VENV_VIRTUALENV_DIR)/chpl-test-reqs
 CHPL_VENV_CHPLSPELL_REQS=$(CHPL_VENV_VIRTUALENV_DIR)/chpl-chplspell-reqs
+
+CHPL_VENV_PIP_CACHE_DIR=$(CHPL_VENV_INSTALL_SUBDIR)/pip-cache
+CHPL_VENV_PIP_FLAGS=--cache-dir $(CHPL_VENV_PIP_CACHE_DIR)
```

Comment:
It would be nice to append this to `CHPL_PIP_INSTALL_PARAMS`, to remove redundancy between these 2 variables.